srcpkgs/gamemode/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'gamemode'
+pkgname=gamemode
+version=1.5.1
+revision=1
+build_style=meson
+configure_args="-Dwith-examples=false -Dwith-systemd=false -Dwith-dbus-service-dir=/usr/share/dbus-1/services --prefix=/usr --buildtype debugoptimized"
+hostmakedepends="git cmake meson ninja pkg-config"
```

Comment:
`meson` and ``ninja` don't need to be here. What is `git` used for? You probably want to add versioned tarballs in `distfiles`.